lib/rest_in_peace.rb

```diff
@@ -47,10 +47,8 @@ def clear_changes
   end
 
   def update_attributes(attributes)
-    attributes.each do |key, value|
-      next unless respond_to?("#{key}=")
-      send("#{key}=", value)
-    end
+    warn 'DEPRECATION WARNING: the method update_attributes is deprecated, use method update.'
```

Comment:
Now that you moved it to `active_model_api`, why deprecate this? We certainly need it for classes that should not be ActiveModel compatible.